src/config/dynamic-theme-fixes.config

```diff
@@ -17070,6 +17070,18 @@ INVERT
 
 ================================
 
+zombo.com
+
+CSS
+body {
+    background: none !important;
+}
+img.thefade {
```

Comment:
Why should the opacity be lowered?

Reply:
The colors are very light, that's why I thought they should only be partially visible. To accomplish this in dark mode, I lowered the opacity a bit

Reply:
Well, the colors of the logo are more light than the image, but that could also be because of my monitor settings.